kernel/include/arch/x86/debug.h

```diff
@@ -15,6 +15,13 @@
 #ifndef KERNEL_DEBUG_H
 #define KERNEL_DEBUG_H
 
+#include <elf/elf.h>
+
 void debug_vaddr_identify(lvaddr_t pml4, lvaddr_t vaddr);
 
+void debug_sort_dynsyms (struct Elf64_Sym *dynsyms, int n);
+void debug_setup_stackwalker (uint64_t stack_top, uint64_t stack_bottom, uint64_t text_start, uint64_t text_end, struct Elf64_Sym *dynsyms, char *dynstr, int nsyms);
```

Comment:
It's better to use more specific types instead of `uint64_t`, for example `lpaddr_t` for local physical addresses or `lvaddr_t` for virtual addresses. We might not do it correctly all the time but at least it gives a hint on how that address could be accessed.